### DISCUSSION TITLE
va-tree-view updates giving errors while npm install Module not found: Error: Default condition should be last one

### DISCUSSION THREAD

**Discussion by kumarimanjari** [epicmaxco/vuestic-ui#3605] [General]
Hello @m0ksem,

As suggested, I am opening a discussion about the error I am getting with updating the vuestic-ui version.
I am using npm as package manager and when I try to run the project locally using npm run serve I get the following error - 

Error `**Module not found: Error: Default condition should be last one**`

**Package Manager - npm
Vuestic-ui version trials - 1.7.0 and 1.7.2
Error occurrence - npm run serve**

Please let me know if you need more information

**Comment by m0ksem** [comment]
Can you share your package json dependencies? Both dependencies and devDependencies.

**Comment by kumarimanjari** [comment]
Sure, 
 Dependencies are - 
 ```
    "axios": "^1.2.3",
    "buffer": "^6.0.3",
    "core-js": "^3.8.3",
    "dompurify": "^3.0.3",
    "jest-junit": "^15.0.0",
    "jest-sonar": "^0.2.15",
    "pinia": "^2.0.33",
    "vue": "^3.2.13",
    "vue-i18n": "^9.3.0-beta.12",
    "vue-resizable": "^2.1.7",
    "vue-router": "^4.1.6",
    "vuestic-ui": "1.7.0"
```
Dev Dependencies  are - 
    ```
    "@babel/core": "^7.12.16",
    "@babel/eslint-parser": "^7.12.16",
    "@pinia/testing": "^0.0.15",
    "@vue/cli-plugin-babel": "~5.0.0",
    "@vue/cli-plugin-eslint": "~5.0.0",
    "@vue/cli-plugin-unit-jest": "~5.0.0",
    "@vue/cli-service": "~5.0.0",
    "@vue/test-utils": "^2.2.6",
    "@vue/vue3-jest": "^27.0.0",
    "babel-jest": "^27.5.1",
    "babel-plugin-transform-require-context": "^0.1.1",
    "bamboo-jest-reporter": "^1.0.3",
    "copy-webpack-plugin": "^11.0.0",
    "eslint": "^7.32.0",
    "eslint-plugin-vue": "^8.0.3",
    "jest": "^27.0.5",
    "jest-junit": "^15.0.0",
    "jest-sonar-reporter": "^2.0.0",
    "sonar-scanner": "^3.1.0"
```

**Comment by kumarimanjari** [comment]
@m0ksem - Do you have any updates for me, or you need any information on this

**Reply by m0ksem** [reply]
Hi @codeonmoon,  for webpack go to `node_modules/vuestic-ui/package.json` and change:
```json
"exports": {
    ".": {
      "import": {
        "node": "./dist/esm-node/main.mjs",
        "default": "./dist/es/main.js",
        "types": "./dist/types/main.d.ts"
      },
      "require": "./dist/cjs/main.js"
    },
    "./css": {
      "import": {
        "node": "./dist/vuestic-ui.css",
        "default": "./dist/styles/index.css"
      }
    },
    "./web-components": "./dist/web-components/main.js",
    "./iife": "./dist/iife/main.js",
    "./types": "./dist/types/main.d.ts",
    "./styles/*": "./dist/styles/*",
    "./dist/*": "./dist/*"
  },
```

to

```json
"exports": {
    ".": {
      "import": {
        "node": "./dist/esm-node/main.mjs",
        "types": "./dist/types/main.d.ts",
        "default": "./dist/es/main.js"
      },
      "require": "./dist/cjs/main.js"
    },
    "./css": {
      "import": {
        "node": "./dist/vuestic-ui.css",
        "default": "./dist/styles/index.css"
      }
    },
    "./web-components": "./dist/web-components/main.js",
    "./iife": "./dist/iife/main.js",
    "./types": "./dist/types/main.d.ts",
    "./styles/*": "./dist/styles/*",
    "./dist/*": "./dist/*"
  },
```

**Reply by kumarimanjari** [reply]
@m0ksem - Thank you for the updates.
Here are some findings from my side when version was 1.7.0, npm run serve worked somehow without updating the package.json which you mentioned here.
With v1.7.2, I did the changes and then only npm run serve worked. 
But at the end it is working.

But whatever related to tree-view is added here - https://github.com/epicmaxco/vuestic-ui/releases/tag/v1.7.0 I am not able to use them in my implementation. How to use them?

**Reply by m0ksem** [reply]
What exactly doesn't work for you?

**Reply by kumarimanjari** [reply]
@m0ksem - Below are the points which is not working or may be I am not able to test it.
Also sorry for the late response. 
Could you please let me know how to test this. I tested this also on the official website and don't see any cursor on mouse hover over labels of child and parent

feat(VaTreeView): add selected node class by @mengqiuleo in https://github.com/epicmaxco/vuestic-ui/pull/3464
feat(VaTreeView): add cursor style when mouse moves over node by @mengqiuleo in https://github.com/epicmaxco/vuestic-ui/pull/3473

**Reply by m0ksem** [reply]
@codeonmoon. If I understood correctly, you need this behavior: https://codesandbox.io/p/sandbox/thirsty-violet-4pqxpp?file=/src/App.vue:8,19

**Reply by kumarimanjari** [reply]
@m0ksem - Yes, I am talking about this only  👍 . It is amazing, but I found that once we are clicking on **Action parent folder** and then selecting Platform Games, this **'Platform Games'** is not getting selected meaning a background color is not getting added.
Also how it is working here and not in my project inspite of the fact that I am also using v1.7.2
What could be wrong from my side?
